paper/JMNotes.Rmd

```diff
@@ -0,0 +1,45 @@
+# remarks JM
+
+- abstract: it is at least uncommon to cite other works in the abstract.
+If not explicitly required by the journal, I would refrain from it since you most likely cite all these works in the paper itself.
+- The last sentence of the abstract does not add much.
+Delete it or rewrite in a more compelling way.
+
+- I like the crop classification example, however, it is not entirely clear to me what agricultural districts are. 
+And though the crop classification introduces non-standard spatial resampling strategies, it does not make reference to the more common spatial resampling procedure (k-means clustering) where clustered coordinates are considered, which might be a more gentle introduction to spatial CV for readers who are not already familiar with the concepts.
+But I might be wrong.
+Looking at Table 1, the spatial blocking/tiles approach seems to be even more popular than the kmeans clustering approach.
+
+- Table 1: Wow, not too many papers have so far used spatial CV.
+Is this a comprehensive list?
+Have you somewhere listed the references by chosen strategy?
+If so, put it in the Appendix and reference the Appendix in the caption.
+Not entirely clear, what the footnotes refer to.
+These seem to be studies which have used the corresponding spatial CV approach but it is not all of them (e.g., spcv_block has been used by 28 papers but only 3 are referenced). 
+Please clarify.
+
+- I really do like the visualizations, great job! And they help the reader a lot to better grasp the differences between the resampling strategies.
+
+- In case, you didn't know: from 1.0.0 onwards, sf uses s2 by default, i.e. spherical geometries.
+Hence, I am not sure if there is really much more work to do (see https://r-spatial.github.io/sf/articles/sf7.html).
+I am referring to this sentence in the paper:
+
+>It is planned to provide a more consistent interface that supports all CRS and allows the use of distance measures in projected planar space or using great circle distances on the unprojected ellipsoid, in the case of unprojected CRS.
```

Comment:
s2 geometries, i.e., computations on the sphere, are now used by **sf** for geographic (i.e. unprojected) coordinates (e.g., CRS:4326) by default. But this also means that as long as you provide a projected CRS, computations are still done on the plane. This again would mean that there might be more work to do if you really intended to provide a consistend interface that supports all CRS. This is just for your information. In any case, you should rewrite the sentence if you use **sf** under the hood for all geometric operations by replacing `using great circle distances on the unprojected ellipsoid, in the case of unprojected CRS` with `using spherical distances in the case of unprojected CRS`.

Reply:
Thanks!

I am not sure if I want/can provide a consistent interface here given all the different underlying approaches and the substantial impact such a change would bring.
I really fear that users might get an unwanted result because they are not aware of this option/what it does and that it might come with more issues than it actually helps.

For now I've removed the sentence to avoid false promises.